modules/tls-profiles-kubernetes-configuring.adoc

```diff
@@ -0,0 +1,122 @@
+// Module included in the following assemblies:
+//
+// * security/tls-profiles.adoc
+
+[id="tls-profiles-kubernetes-configuring_{context}"]
+= Configuring the TLS security profile for the Kubernetes control plane
+
+To configure a TLS security profile for the Kubernetes control plane, edit the `APIServer` custom resource (CR).  If a TLS profile is not configured, the default TLS profile is `Intermediate`. The Kubernetes control plane consists of the Kubernetes API server, the Kubernetes scheduler, and the Kubernetes controller manager.
+
+.Sample `APIServer` CR that configures the `Old` TLS security profile
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: APIServer
+ ...
+spec:
+  tlsSecurityProfile:
+    old: {}
+    type: Old
+ ...
+----
+
+The TLS security profile defines the the minimum TLS version and the TLS ciphers required by the Kubernetes control plane components to communicate with each other.
+
+You can see the ciphers and the minimum TLS version of the configured TLS security profile  in the `APIServer` custom resource (CR) under `Spec.Tls Security Profile` and `Status.Tls Profile`. 
+
+[IMPORTANT]
+====
+The Kubernetes control plane does not support TLS `1.3` and because the `Modern` profile requires TLS `1.3` it is not supported. The Kubernetes control plane converts the `Modern` profile to `Intermediate`.
+
+The Kubernetes control plane also converts the TLS `1.0` of an `Old` or `Custom` profile to `1.1`, and TLS `1.3` of a `Custom` profile to `1.2`.
+====
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Edit the default `APIServer` CR to configure the TLS security profile:
++
+[source,terminal]
+----
+$ oc edit APIServer cluster
+----
++
+.Sample `tls-security-profile.yaml` file
```

Comment:
Since you changed this to use `oc edit`, there isn't this `tls-security-profile.yaml` file anymore.

Also, right now this is just a single step. Could you break it up more info separate steps? Like:

```
. Edit the default `APIServer` CR to configure the TLS security profile:
// command
. Add the `spec.tlsSecurityProfile` field:
// YAML details
. Save the file to apply the changes.
```

Same for the ingress procedure

Reply:
For the record, I changed to `oc edit` as the APIServer CR already exists. Good point to add the additional steps.